### PR TITLE
[UX] Graphic Design: Create a Layer5-branded 3D book to represent the Community Handbook

### LINKED ISSUE #4075: [UX] Graphic Design: Create a Layer5-branded 3D book to represent the Community Handbook
### Current Behavior
The Community Handbook has a number of pages and is commonly referenced by community members. 

### Desired Behavior
It would be nice to have a visual representation of this handbook to be displayed on pages like the [Community Managers page
](https://layer5.io/community/community-managers).

Screenshot 2023-04-07 at 5 49 47 PM

### Screenshots / Mockups
An example of such a 3D book can be seen in the "Cloud native focused" section of the layer5.io home page.

### Implementation
1. Create a graphic using elements of Layer5's brand (see the community shared drive for many, many assets)
2. Use https://3dbookcovergenerator.netlify.app to style that graphic as a 3D book.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Savio629**
**Description**
[UX] Graphic Design: Created a Layer5-branded 3D book to represent the Community Handbook

This PR fixes #4075 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d29b4b1006409f7c5be886ba7e6faec7a92b9211 at: https://648d69ff73a33f6183c9dff1--layer5.netlify.app

**Comment by Savio629** [comment]
chlayer

@thisiskaransgit @vishalvivekm 
Do we need light design for dark mode?
Because the mentioned light design for dark mode doesn't fit well with the theme - #4075

**Comment by thisiskaransgit** [comment]
@Savio629,  The current change looks good to me. Better if we discuss it at the website meeting.

File: src/sections/Community/Handbook/BookComponent/BookComponent.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import BookComponentWrapper from "./BookComponent.style";
+
+const BookComponent = () => {
+  return (
+    <BookComponentWrapper>
+    <a
```
Comment (leecalcote) [inline src/sections/Community/Handbook/BookComponent/BookComponent.js:7]: Are we able to use a `<Link>` here? Use Link for site internal links like this one where possible.

File: src/sections/Community/Handbook/BookComponent/BookComponent.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      className="book-container"
+      href="https://layer5.io/community/handbook"
+      target="_blank"
+      rel="noreferrer noopener"
+    >
+      <div className="book">
+        <img
+          alt="Community HandBook"
+          src="https://user-images.githubusercontent.com/91362589/245493993-1780f4f9-b332-4b91-95ac-841013624b86.png"
```
Comment (leecalcote) [inline src/sections/Community/Handbook/BookComponent/BookComponent.js:16]: Do not use an external reference like this. Use a site-local image.

Comment (leecalcote) [reply src/sections/Community/Handbook/BookComponent/BookComponent.js:16]: Use SVGs, not PNGs, please.

**Review by leecalcote — CHANGES_REQUESTED** [review]
@randychilau how does this PR fit / not fit with our approach to light vs dark mode flicker prevention?

**Comment by randychilau** [comment]
> @randychilau how does this PR fit / not fit with our approach to light vs dark mode flicker prevention?

Hi @leecalcote,

I reviewed this PR and didn't notice any flicker issues with regards to light vs. dark mode  (even when refreshing):

[3dbook2.webm](https://github.com/layer5io/layer5/assets/90356410/ab010e8a-4860-4ac5-8bf9-d52427bbc66e)

However I did notice that the book image can take a moment to load when doing a "hard refresh", momentarily presenting a blank book while the image loaded:

[3dbook3.webm](https://github.com/layer5io/layer5/assets/90356410/956b1070-0e2f-404d-a558-4efe66ad1192)

This can be due to a couple of factors:
- size of the image file 
  image
  - due to image dimensions
  - file format
- image being stored outside of the site and remotely accessed (as Lee mentioned)

Potential changes to resolve the issue:
- resize the file to be smaller dimensions (keeping aspect ratio)
- change the file format from png to either SVG or webp
- change the file location to be stored in the site

Hope that helps.

Cheers,
Randy

**Comment by Savio629** [comment]
Was going to convert the png into webp and then use it as a site-local image but forgot to mentioned it her...

**Comment by l5io** [comment]
🚀 Preview for commit d2cc5610638214af48b99666fa09ea57b026fe09 at: https://648f6eed2348aa6d36ce6335--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@Savio629 Let's discuss this on websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would :)

**Comment by Savio629** [comment]
@Chadha93 
Added rotation on reload and decreased a little bit of shadow...
https://github.com/layer5io/layer5/assets/91362589/31cac91b-c395-48f2-bfb1-4f5d84390e03

**Comment by leecalcote** [comment]
@Savio629 it seems that we should get this book on the /community page and/or the /handbook page, too.

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Looks great!

**Comment by Savio629** [comment]
> @Savio629 it seems that we should get this book on the /community page and/or the /handbook page, too.

As it is redirecting to handbook page itself we can maybe add it to community page...
I'll add it in website's meeting agenda.

**Comment by leecalcote** [comment]
Sounds real good.

**Pull request merged**